README.md

```diff
@@ -9,18 +9,19 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 ## Example
 
 ```js
+/* The superagent-use module returns a clone of the superagent provided with the new functionality. */
+var agent = require('superagent-use')(require('superagent')),
+    prefix = require('superagent-prefix'); /* A sample superagent plugin/middleware. */
```

Comment:
Here's another one, also while at it, place the comment about the sample above the variable declaration.